beets/autotag/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 """Facilities for automatically determining files' correct metadata.
 """
-import logging
+from beets import logging
 
 from beets import config
```

Comment:
Huh. Or maybe not, judging by other places in the code. There should be a blank line between the declarations though.